docs/digger/developer/jenkinsfile-fastlane.asciidoc

```diff
@@ -0,0 +1,5 @@
+== Fastlane Support
+
```

Comment:
@laurafitzgerald  might be an idea to link to https://fastlane.tools/ ?

Reply:
@finp done :)